definitions/reports/advisor_on_prem_remediations.rb

```diff
@@ -0,0 +1,25 @@
+module Reports
+  class AdvisorOnPremRemediations < ForemanMaintain::Report
+    metadata do
+      description 'Report number of remediations based on advisor rules from advisor on premise'
+      confine do
+        feature(:installer)&.answers&.dig('foreman::plugin::rh_cloud', 'enable_iop_advisor_engine')
+      end
+    end
+
+    def run
+      data_field('advisor_on_prem_remediations_count') { advisor_on_prem_remediations_count }
+    end
+
+    def advisor_on_prem_remediations_count
+      sql_count('(
+        SELECT tinv.host_id, jobs.id
+        FROM job_invocations AS jobs
+        INNER JOIN remote_execution_features AS rexf ON jobs.remote_execution_feature_id = rexf.id
+        INNER JOIN template_invocations AS tinv ON jobs.id = tinv.job_invocation_id
+        WHERE rexf.label = \'rh_cloud_remediate_hosts\'
+        AND tinv.host_id IS NOT NULL
+      ) AS subquery')
```

Comment:
This should give the same results
```suggestion
      sql_count('
        job_invocations AS jobs
        INNER JOIN remote_execution_features AS rexf ON jobs.remote_execution_feature_id = rexf.id
        INNER JOIN template_invocations AS tinv ON jobs.id = tinv.job_invocation_id
        WHERE rexf.label = \'rh_cloud_remediate_hosts\'
        AND tinv.host_id IS NOT NULL
      ')
```

Reply:
I confirm that it gives the same result in my testing case. Applied.